### ISSUE TITLE
Cannot edit/manage single currencies in Multi-Currency

### ISSUE THREAD

**Issue by jessepearson** [Automattic/woocommerce-payments#4042]
### Describe the bug
If you go to WooCommerce > Settings > Multi-Currency, and then click to _manage_ a single currency, the page does not load. This error is shown in the console:

```
Uncaught TypeError: Cannot read properties of undefined (reading 'country')
```

This was introduced in #3971 and happens because `wcpaySettings` is not loaded into Multi-Currency pages. Discussion about this is here: p1648482499097799-slack-C01B8KNUYSW

### To Reproduce

1. Navigate to WooCommerce > Settings > Multi-Currency.
2. If you do not have any additional currencies, click _Add currencies_ and add a currency.
3. Click _manage_ to update a single currency's settings.
4. Page errors out.

**Comment by amElnagdy** [comment]
Reported in 4897206-zen

**Comment by htdat** [comment]
Closing this issue as it was fixed in WCPay 3.9.1. See https://github.com/Automattic/woocommerce-payments/pull/4053, which was then merged back to `develop`.
